### PR TITLE
remove name for default drop acl in networkpolicy

### LINKED ISSUE #1445: networkpolicy does not work when policy‘s name length is more than 63 characters
# Expected Behavior
create a networkpolicy, with normal ingress or egress acl, the name is more than 63 characters, expect the networkpolicy create successfully and work normally.

# Actual Behavior
the networkpolicy can be created successfully but does not work

# Steps to Reproduce the Problem

1.  create networkpolicy with name length more than 63 characters 
2. test if the networkpolicy works

# Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

  ```
  (paste your output here)
  ```

- kube-ovn version:
  
  ```
  v1.7.3
  ```

- operation-system/kernel version:
  
  **Output of `awk -F '=' '/PRETTY_NAME/ { print $2 }' /etc/os-release`:**
  **Output of `uname -r`:**
  ```
  (paste your output here)
  ```
wecom-temp-470d1633beb67b5107cc39a0266f290f

### PR DESCRIPTION
**Pull request by hongzhen-ma**
#### What type of this PR
- Bug fixes

####
remove name for default drop acl in networkpolicy

#### Which issue(s) this PR fixes:
Fixes #1445

### CODE REVIEW

File: pkg/ovs/ovn-nbctl.go [hunk 1]
```diff
@@ -1569,6 +1569,9 @@ func (c Client) CreateNpAddressSet(asName, npNamespace, npName, direction string
 
 func (c Client) CreateIngressACL(npName, pgName, asIngressName, asExceptName, svcAsName, protocol string, npp []netv1.NetworkPolicyPort) error {
 	var allowArgs []string
+	if len(npName) > 63 {
```
Comment (oilbeater) [inline pkg/ovs/ovn-nbctl.go:1572]: the  name may conflict with each other, we should consider using hash or uuid

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: pkg/ovs/ovn-nbctl.go (hunk 1 above)
Comment (hongzhen-ma) [reply pkg/ovs/ovn-nbctl.go:1573]: modified

Comment (oilbeater) [reply pkg/ovs/ovn-nbctl.go:1573]: what about delete and update

Comment (hongzhen-ma) [reply pkg/ovs/ovn-nbctl.go:1573]: The name is only used for default drop acl with priority equals 2000 and the name is never used later.
All the processes about acls later is based on port-group name, including create acls with other priority、update and delete.
Maybe it's better to delete the acl name.

**Review by oilbeater — APPROVED** [review]

**Comment by hongzhen-ma** [comment]
已回合release-1.8和release-1.9分支

**Pull request merged**